x-pack/plugins/advanced_ui_actions/public/components/action_wizard/action_wizard.tsx

```diff
@@ -65,6 +48,11 @@ export interface ActionWizardProps {
    * config changed
    */
   onConfigChange: (config: ActionBaseConfig) => void;
+
+  /**
+   * Context will be passed into ActionFactory's methods
+   */
+  context: ActionFactoryBaseContext;
```

Comment:
I vote we scratch factory context until we actually need it.

Reply:
See comments about `placeContext` above.

We can scratch it, but then the Drilldown UI flyout would not know from where it was open. And Dashboard-to-Dashboard would not know the current panel title and current dashboard ID. I think we need it, let's discuss it in our sync today.